### PR TITLE
Release 3.5.0

### PR DESCRIPTION
**Pull request by JeroenDeDauw**

### CODE REVIEW

**Comment by JeroenDeDauw** [comment]
Want to do a fluffness check @kghbln? :)

File: INSTALL.md [hunk 1]
```diff
@@ -12,55 +12,61 @@ These are the installation and configuration instructions for the [Maps extensio
...
 	<tr>
-		<th><a href="https://github.com/JeroenDeDauw/Maps/blob/master/docs/RELEASE-NOTES.md">Maps 3.4.1</a></th>
+		<th><a href="RELEASE-NOTES.md">Maps 3.5.x</a></th>
+		<td>Stable release</td>
+		<td>2016-04-01</td>
+		<td><a href="https://github.com/JeroenDeDauw/Maps/tree/3.5.0">3.5.0</a></td>
+	</tr>
+	<tr>
+		<th><a href="RELEASE-NOTES.md#maps-341">Maps 3.4.1</a></th>
 		<td>Stable release</td>
```
Comment (kghbln) [inline INSTALL.md]: That should be "Obsolete release"

File: README.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ Features:
...
 (https://www.mediawiki.org/wiki/Extension:Semantic_Maps).
 
-View the [release notes](docs/RELEASE-NOTES.md) for recent changes to Maps.
+View the [release notes](RELEASE-NOTES.md) for recent changes to Maps.
 
 ### User manual
 
-* [Installation and configuration](docs/INSTALL.md)
+* [Installation and configuration](INSTALL.md)
 * [Usage instructions and examples](https://semantic-mediawiki.org/wiki/Maps)
```
Comment (kghbln) [inline README.md]: Could you switch to https://www.sema....

**Comment by kghbln** [comment]
> Want to do a fluffness check @kghbln? :)

Done with two comments. The rest seems fluffy to me.

**Comment by kghbln** [comment]
@JeroenDeDauw We need a MW 1.26+ fix here too.

**Comment by kghbln** [comment]
Done with https://github.com/JeroenDeDauw/Maps/pull/121

**Comment by kghbln** [comment]
@JeroenDeDauw Thanks for fluff!

**Pull request merged**